### DISCUSSION TITLE
Woopayments doesn't support blocks?

### DISCUSSION THREAD

**Discussion by turleybw** [woocommerce/woocommerce#43594] [Checkout Flow]
I want to customize my cart using blocks in the 2024 theme, but after lengthy setup of WooPayments I find blocks aren't supported. This is really frustrating. What's the plan and timeline for support?

**Comment by senadir** [comment]
I'm not really sure I understand, WooPayments was the first plugin to support Cart/Checkout blocks, where are you seeing this?
